hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/converter/MessageConverterFactory.java

```diff
@@ -0,0 +1,19 @@
+package pl.allegro.tech.hermes.consumers.consumer.converter;
+
+import org.apache.avro.Schema;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.message.converter.AvroToJsonConverter;
+
+public class MessageConverterFactory {
+
+    private final NoOperationMessageConverter noOperationMessageConverter = new NoOperationMessageConverter();
+
+    public MessageConverter create(Topic.ContentType contentType, String messageSchema) {
+        switch(contentType) {
+            case AVRO:
+                return new AvroToJsonMessageConverter(new AvroToJsonConverter(new Schema.Parser().parse(messageSchema)));
```

Comment:
Its factory, so I prefer creating objects here instead of some constructor. Also this approach simplifies witting clean unit tests.